src/Input/Textarea.js

```diff
@@ -190,6 +190,7 @@ class Textarea extends Component {
           onChange={this.handleChange}
           defaultValue={defaultValue}
           value={value}
+          disabled={disabled}
```

Comment:
That's exactly what I was thinking of. I would rather have less code here. Thanks